### ISSUE TITLE
Configuring Multi-factor Authentication Error

### ISSUE THREAD

**Issue by SkillableKA** [MicrosoftLearning/MD-102T00-Microsoft-365-Endpoint-Administrator#90]
Practice Lab 0501: Configuring Multi-factor Authentication

Task 2: Enable MFA for a user

Step: 7. In the multi-factor authentication page, select service settings. Select Allow users to remember multi-factor authentication on devices they trust.

Description of issue:

When users are attempting to access the 'Additional cloud-based multifactor authentication settings' [P1] and then 'Service settings' [P2] they are met with a service connection error and are unable to proceed [P3].

[P1]
![image](https://github.com/user-attachments/assets/c182a2b7-2f84-4ef1-a52b-4b98d525518a)
[P2]
![image](https://github.com/user-attachments/assets/8dca8a43-bf59-4c8b-8362-17bcd8a3962e)
[P3]
![image](https://github.com/user-attachments/assets/f52789e0-0898-4201-843c-f58622b34f7a)

**Comment by MSFT-rstearman** [comment]
I will look into this and get back to you. Thanks for letting me know.
________________________________
From: SkillableKA ***@***.***>
Sent: Thursday, September 5, 2024 6:32 AM
To: MicrosoftLearning/MD-102T00-Microsoft-365-Endpoint-Administrator ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [MicrosoftLearning/MD-102T00-Microsoft-365-Endpoint-Administrator] Configuring Multi-factor Authentication Error (Issue #90)


Practice Lab 0501: Configuring Multi-factor Authentication

Task 2: Enable MFA for a user

Step: 7. In the multi-factor authentication page, select service settings. Select Allow users to remember multi-factor authentication on devices they trust.

Description of issue:

When users are attempting to access the 'Additional cloud-based multifactor authentication settings' [P1] and then 'Service settings' [P2] they are met with a service connection error and are unable to proceed [P3].

[P1]
image.png (view on web)<https://github.com/user-attachments/assets/c182a2b7-2f84-4ef1-a52b-4b98d525518a>
[P2]
image.png (view on web)<https://github.com/user-attachments/assets/8dca8a43-bf59-4c8b-8362-17bcd8a3962e>
[P3]
image.png (view on web)<https://github.com/user-attachments/assets/f52789e0-0898-4201-843c-f58622b34f7a>

**Comment by HashirSajjad** [comment]
I am also facing similar issue. Please leave an update if there is any.

**Comment by WigF1** [comment]
It's because of the 'new' interface. It does not currently affect all tenants, at least my delivery last week only affected 3 out of 12 tenants. 

If you use the 'legacy' link (that is listed in the notification displayed in P2 above) to modify any setting and save, the 'new' links work like a charm. 

The legacy link just takes you here : https://account.activedirectory.windowsazure.com/UserManagement/MultifactorVerification.aspx

If I had to guess I'd say the 'new' portal can't handle rehydration of custom settings, or a null set; hence the service connection error.

**Comment by HashirSajjad** [comment]
FYI, I just gave it another try and it worked as it is expected to.

**Comment by KashifKhaan** [comment]
I have faced the same issue, please let us know if it's resolved.

**Comment by SkillableKA** [comment]
Hello, 

Just seeing if there had been any updates to this as we are still receiving tickets with both instructors and students experiencing the issue above. The page will now load, but users are unable to actually save the MFA settings. [P1]

As Steve pointed out, the legacy link will allow you to save the required settings. You can then jump back over to the Entra GUI and the changes will be showing. [P2]

[P1]
![image](https://github.com/user-attachments/assets/109916e5-c26a-4073-8ce8-58cdf34adf9c)

[P2]
![image](https://github.com/user-attachments/assets/210667aa-5507-4299-bb68-68690374465b)

**Comment by WigF1** [comment]
Yeah, I had to advise students last week to use the legacy link to set the minutes. Once saved, it's all gravy, you can then change it in the 'new' page or the 'legacy'.

It's more of a platform issue rather than a lab issue, though. 

I see two choices. We either (a) advise students if it fails to use the legacy link or (b) change the lab steps to always use 'legacy' link. 

I think it might be better to go for option (a), personally.

**Comment by WigF1** [comment]
I just ran through the lab while I was making changes for a PR and it worked like a charm from the 'new' page. @SkillableKA any chance you guys wanna spin up a lab instance and check if you can also set the 30 and save?
